zcash_client_sqlite/src/wallet/init/migrations.rs

```diff
@@ -116,9 +116,10 @@ pub(super) fn all_migrations<
     //                              \      ensure_default_transparent_address    /
     //                               \                     |                    /
     //                                `---- fix_transparent_received_outputs --'
-    //                                        /            |            \
-    //             support_zcashd_wallet_import            |            fix_v_transactions_expired_unmined
-    //                                         v_tx_outputs_return_addrs
+    //                                        /                         \
+    //                     support_zcashd_wallet_import          fix_v_transactions_expired_unmined
+    //                                                                           |
+    //                                                                v_tx_outputs_return_addrs
```

Comment:
In the commit message, @nuttycom says that `fix_v_transactions_expired_unmined` modifies `v_tx_outputs`. AFAICT it does not, and these were fine to be in parallel. However, there's no harm in this graph reorganisation, because `v_tx_outputs_return_addrs` has not yet been released (whereas `fix_v_transactions_expired_unmined` has).